packages/core/examples/meshLoader/index.ts

```diff
@@ -0,0 +1,317 @@
+import type { Types } from '@cornerstonejs/core';
+import {
+  RenderingEngine,
+  Enums,
+  init as csRenderInit,
+  geometryLoader,
+  cornerstoneMeshLoader,
+  eventTarget,
+} from '@cornerstonejs/core';
+import { setTitleAndDescription } from '../../../../utils/demo/helpers';
+import { init as csToolsInit } from '@cornerstonejs/tools';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+geometryLoader.setOptions({
+  beforeSend(xhr) {
+    const headers = {
+      Authorization: 'Bearer YOUR_API_KEY',
+    };
+    return headers;
+  },
+});
+
+geometryLoader.registerGeometryLoader('mesh', cornerstoneMeshLoader);
```

Comment:
I think we can just self-register it, like other loaders with automatic registration. If someone has another custom mesh loader, they can register it with higher priority.